src/workerd/api/global-scope.c++

```diff
@@ -364,6 +364,7 @@ void ServiceWorkerGlobalScope::startScheduled(
 
 kj::Promise<WorkerInterface::AlarmResult> ServiceWorkerGlobalScope::runAlarm(
     kj::Date scheduledTime,
+    kj::Duration timeoutMs,
```

Comment:
You'll probably hate me for saying this, but if you have a `kj::Duration`, the name should be `timeout` since the Duration handles units for you.  Please fix!